resources/education.md

```diff
@@ -74,7 +74,7 @@ This pre-recorded course is designed for developers with intermediate knowledge
 
 ## Localization Institute
 
-The **Machine Translation Master Class** is offered by the Localization Institute and taught by Peng Wang.
+The **Machine Translation Master Class** is offered by the Localisation Institute and taught by Peng Wang.
```

Comment:
I'm not sure about changing an organization name to conform with UK spelling.

Reply:
Right, we should not